docs/src/golive/steps/subdomains.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Subdomains across different projects"
+sidebarTitle: "Subdomain handling"
+weight: 2
+description: "How to handle multiple subdomains on different projects."
+---
+
+If you want to host multiple projects off of the same domain name, such as `foo.example.com` and `bar.example.com`, you will need to register an additional DNS record to make that possible.  If you do not, the second project you try to create will reject your domain.
+
+## Quick solution
+
+To enable multiple projects to use subdomains of the same domain, you must add a DNS `TXT` record with your DNS registrar.  Consult your registrar's documentation for how to do so, as it varies from one registrar to another.
+
+The record should look like the following:
+
+```
+_public-suffix-root.example.com TXT "public-suffix-root=example.com"
+```
+
+Replace `example.com` with your actual domain name.  That will tell Platform.sh that `example.com` is a top-level domain, and allow you to add multiple `something.example.com` domains on different projects on Platform.sh.
```

Comment:
Maybe specify "if you want to add example.com after adding foo.example.com on another project, you will need to ask our staff in a support ticket" or something along those lines?